picard/log.py

```diff
@@ -113,7 +113,7 @@ def formatted_log_line(level, time, message, timefmt='hh:mm:ss',
 
 def _stderr_receiver(level, time, msg):
     try:
-        sys.stderr.write(formatted_log_line(level, time, msg + os.linesep))
+        sys.stderr.write("%r%s" % (formatted_log_line(level, time, msg), os.linesep))
```

Comment:
It will ouput the whole line as 'u"..."', not what we want (at all)

```
u"D: 14:23:36 WSREQ: Last request to (u'musicbrainz.org', 80) was 1000 ms ago, starting another one"
```
The issue is related to encodings and win console (and perhaps others), and we are using unicode internally, the exception you get is because you output to a console with another encoding and not all chars can be converted to the destination charmap.

See:
- http://stackoverflow.com/questions/5419/python-unicode-and-the-windows-console
- https://wiki.python.org/moin/PrintFails
- https://www.python.org/dev/peps/pep-0528/
- http://stackoverflow.com/questions/1545263/utf-8-in-python-logging-how

Can you try without this PR changes, but adding line:
```
sys.stderr = codecs.getwriter(locale.getpreferredencoding())(sys.stderr)
```
just before `sys.stderr.write(...)` call ?